models/ac_generator.h

```diff
@@ -128,12 +164,20 @@ class ac_generator : public Node
     double A_11_;
   };
 
+  double
+  get_I_() const
+  {
+    return S_.I_;
```

Comment:
@appukuttan-shailesh I am sorry, I had not thought about the fact that the device might be inactive. To account for that, we need `S_.I_`.  It might be a good idea to explain this in the doxygen comment for that variable.